test/common/http/metadata_interface_test.cc

```diff
@@ -0,0 +1,53 @@
+#include <array>
+#include <cstdint>
+#include <string>
+
+#include "envoy/http/metadata_interface.h"
+
+#include "source/common/common/fmt.h"
+#include "source/common/common/logger.h"
+#include "source/common/http/utility.h"
```

Comment:
This should be: "fmt/ranges.h", and please add a comment why it is added.

Reply:
done